templates/drone/4/README.md

```diff
@@ -0,0 +1,19 @@
+# Drone
+
+### Info:
+
+This template creates an instance of Drone CI server 0.8 along with selectable number of agents to perform the builds.
+
+### Usage:
+
+Select the Drone template from the catalog. Provide the following information:
+
+1. Publish port
+2. Agents scale
+3. Drone secret
+4. Run mode. debug | release
+3. Remote driver and config. (Ie. GitHub)
+4. Database driver and config. (Ie. sqlite)
+
```

Comment:
Please, add a note or warning, about the host label needed to launch lb. I propose you to be able of change this label when deploy. See below